constructor-manager-ui/src/constructor_manager_ui/widgets/dialog.py

```diff
@@ -0,0 +1,465 @@
+        )
+        installation_actions_layout.addWidget(self._revert_button, 2, 0)
+        installation_actions_layout.addWidget(self.revert_label, 2, 1)
+
+        # Reset action
+        self._reset_button = QPushButton("Reset Installation")
+        self.reset_label = QLabel(
+            "Reset the current installation to clear "
+            "preferences, plugins, and other packages"
+        )
+        installation_actions_layout.addWidget(self._reset_button, 3, 0)
+        installation_actions_layout.addWidget(self.reset_label, 3, 1)
+
+        # Uninstall action
+        self.uninstall_button = QPushButton("Uninstall")
+        self.uninstall_button.setObjectName("uninstall_button")
+        self.uninstall_label = QLabel(
+            f"Remove the {self.package_name} Bundled App "
+            "and Installation Manager from your computer"
+        )
+        self.uninstall_button.setVisible(False)
+        self.uninstall_label.setVisible(False)
+        # TODO: to be enabled later on
+        # installation_actions_layout.addWidget(self.uninstall_button, 3, 0)
+        # installation_actions_layout.addWidget(uninstall_label, 3, 1)
+
+        installation_actions_group.setLayout(installation_actions_layout)
+
+        # Signals
+        self._restore_button.clicked.connect(self.restore_installation)
+        self._revert_button.clicked.connect(self.revert_installation)
+        self._reset_button.clicked.connect(self.reset_installation)
+        self.uninstall_button.clicked.connect(self.uninstall)
+
+        return installation_actions_group
+
+    def _create_feedback_group(self):
+        feedback_group = QGroupBox("Feedback")
+        self._status_data = QTextEdit(self)
+        self._status_error = QTextEdit(self)
+
+        self._status_data.setReadOnly(True)
+        self._status_error.setReadOnly(True)
+
+        feedback_layout = QVBoxLayout()
+        feedback_status_layout = QHBoxLayout()
+        feedback_status_layout.addWidget(self._status_data)
+        feedback_status_layout.addWidget(self._status_error)
+
+        feedback_layout.addLayout(feedback_status_layout)
+        feedback_group.setLayout(feedback_layout)
+        return feedback_group
+
+    def setup_layout(self):
+        main_layout = QVBoxLayout(self)
+
+        # Install information
+        install_information_group = self._create_install_information_group()
+        main_layout.addWidget(install_information_group, stretch=2)
+
+        # Packages
+        packages_group = self._create_packages_group()
+        main_layout.addWidget(packages_group, stretch=2)
+
+        # Installation Actions
+        installation_actions_group = self._create_installation_actions_group()
+        main_layout.addWidget(installation_actions_group, stretch=2)
+
+        # Installation Actions
+        feedback_group = self._create_feedback_group()
+        main_layout.addWidget(feedback_group, stretch=1)
+        print(self.log)
```

Comment:
Should this and other `print` statements in the file be transformed into `logger.info`/`logger.debug` calls?

Reply:
Yes absolutely